File: tests/conformance/fixtures/go/06-services.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("test").Run("pytest").
		Service("postgres:15", "db").
		Service("redis:7", "cache")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/06-services.exs
use Sykli

pipeline do
  task "test" do
    run "pytest"
    service "postgres:15", "db"
    service "redis:7", "cache"
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
